platform/ext/target/nordic_nrf/common/native_drivers/spu.h

```diff
@@ -77,4 +77,46 @@ void spu_regions_sram_config_non_secure( uint32_t start_addr, uint32_t limit_add
  */
 void spu_regions_flash_config_non_secure_callable(uint32_t start_addr, uint32_t limit_addr);
 
+/**
+ * \brief Restict access to peripheral to secure
+ *
+ *  Configure a device peripheral to be accessible from Secure domain only.
+ *
+ * \note
+ * - peripheral base address must be a valid peripheral ID
```

Comment:
The peripheral ID is something different than its base address. I think this comment is a bit confusing.

Reply:
Right. So I re-wrote to: "must correspond to valid peripheral ID"